stack-scholar/recipes/deploy.rb

```diff
@@ -1,6 +1,23 @@
 include_recipe 'php-fpm::service'
 include_recipe 'nginx-app::service'
 
+#
+# An attempt to make sure that the php prefix given is the prefix
+# chosen for php-cli at the end of an install
+php_package_prefix = node['php']['ppa']['package_prefix'].gsub(/php/, '')
```

Comment:
@gilleyj there's a thing with `update-alternatives` already in there. But I let you do your thing!